client/src/Pages/Exercise/index.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
+import ExercisesTable from '../../components/exercises/ExercisesTable';
 
 function ExercisePage() {
   return (
     <>
-      <div>
-        <h1>Exercise Page </h1>
+      <div style={{ paddingTop: '100px' }}>
```

Comment:
If it’s not too much trouble, you could move this into css or useStyles. I try to be disciplined and avoid inline styles as much as possible myself, even if a component only has one style property like this, because it separates your code better. Once you add a little bit of inline styles, you’ll soon find yourself adding more and more 😅